src/org/usfirst/frc/team2706/robot/Robot.java

```diff
@@ -80,9 +81,11 @@ public void robotInit() {
                         /* no switch: do nothing */ new ArcadeDriveWithJoystick(),
                        /* position 1: do nothing */ new ArcadeDriveWithJoystick(),
             /* position 2: Move Forward one foot */ new StraightDriveWithEncoders(0.3, 2, 1, 5, "AutoForwardFoot"),
-                                                    new RotateDriveWithGyro(0.5, 90, 5, "AutoTurn90"),
+                                                    new RotateDriveWithGyro(0.5, 90, 5, "AutoTurn900"),
```

Comment:
This should be reverted.

Reply:
Thats your fault for breaking it with config